examples/audio-queue-squarewave.rs

```diff
@@ -36,7 +36,7 @@ fn main() {
         // default sample size 
         };
 
-    let device = audio_subsystem.open_queue::<i16>(None, &desired_spec).unwrap();
+    let device = audio_subsystem.open_queue::<i16, Option<&str>>(None, &desired_spec).unwrap();
```

Comment:
Looks very surprising that you have to do this. What's the error message without the `Option<&str>` ? Having to do this is almost a step backwards IMO.